.travis.yml

```diff
@@ -268,6 +269,9 @@ matrix:
             - pypy
 
 before_install:
+  - if [[ "${TEST_COVERAGE}" == "true" ]]; then
+      pip install codecov coverage;
```

Comment:
coverage is installed into a virtualenv environment, but a conda environment is installed below.